### PR TITLE
feat: Generate RSS feed

### PR DESCRIPTION
**Pull request by jbleuzen**
Hello, 
Totally forgot that we can have discussion here...

Here are my propositions to display rss links on the site.

CleanShot 2023-06-12 at 21 18 42@2x

CleanShot 2023-06-12 at 21 21 07@2x

Maybe a link on the homepage could be nice maybe in the menu ? 
Let me know what you think !

### CODE REVIEW

**Review by j0k3r — DISMISSED** [review]

**Comment by nicosomb** [comment]
Is that possible to keep `https://wallabag.org/en/news.rss` for the feed URL? Or setting a redirection?

**Comment by jbleuzen** [comment]
Yes, I think it's doable !

**Comment by jbleuzen** [comment]
Ok, it's not that easy due to Github Pages limitations...
I have two possible solutions that I can't test : 
 - Create a HTML page that redirect to the new location.
  The issue is I don't know if RSS Reader will handle this redirection correctly and if GH-Pages will allow it to work nicely.
 - Duplicate the rss template file to the old location.
  It's an ugly solution but it'd do the trick.

Maybe we can do some try and fail on the website for the first solution to be sure if what I came up with works ?

File: package-lock.json [hunk 1]
```diff
@@ -1,30 +1,21 @@
 {
     "name": "wallabag",
     "version": "1.0.0",
-    "lockfileVersion": 3,
+    "lockfileVersion": 1,
```
Comment (j0k3r) [inline package-lock.json:4]: What version of NPM/node are you using?
It's weird that the changes is a downgrade version

Comment (jbleuzen) [reply package-lock.json:4]: I'm using v6.14.15. 
That's an old version I know, I've updated it to last version, I'll repush the lock file

Comment (j0k3r) [reply package-lock.json]: Way better!

**Comment by j0k3r** [comment]
About redirection I was wondering if we should deploy the website to Netlify instead? It's more easy to handle redirection.
And the documentation is already deployed using Netlify.

**Comment by nicosomb** [comment]
Propagation to netlify is in progress.

**Comment by jbleuzen** [comment]
Redirections are way easier to setup with Netlify.

**Comment by nicosomb** [comment]
The redirection seems ko

**Comment by jbleuzen** [comment]
Yes it's KO, I forgot to copy it in `_site` !

File: eleventy.config.js [hunk 2]
```diff
@@ -24,7 +25,8 @@ module.exports = function(eleventyConfig) {
   // Copy static files to the root of _site
   eleventyConfig.addPassthroughCopy({
     "./static/": "/",
-    "./CNAME": "/CNAME"
+    "./CNAME": "/CNAME",
```
Comment (j0k3r) [inline eleventy.config.js:28]: This one can be removed

```suggestion
```

**Review by j0k3r — APPROVED** [review]

**Comment by nicosomb** [comment]
Thank you @jbleuzen and @j0k3r !

**Comment by nicosomb** [comment]
The RSS feed and the redirection work ! 🎉 

Capture d’écran 2023-06-15 à 09 24 48

**Comment by jbleuzen** [comment]
Great ! 
I suggested location to place RSS link in first message of this PR but I've not implemented it yet.

If it's OK for you I'll implement it later.

**Comment by nicosomb** [comment]
I thought these changes were embedded with this PR 😉 

OK for the next PR 👍 thank you for your work.

**Pull request merged**